### PR TITLE
docs(authorization): updates casl example in accordance to the changes in v5

### PR DESCRIPTION
**Pull request by stalniy**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[ ] Other... Please describe:
```

## What is the new behavior?

I slightly updated casl example to show how to configure v5 to use classes. The behavior is different in comparison to v4

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: content/security/authorization.md [hunk 1]
```diff
@@ -250,7 +250,10 @@ export class CaslAbilityFactory {
     can(Action.Update, Article, { authorId: user.id });
     cannot(Action.Delete, Article, { isPublished: true });
 
-    return build();
+    return build({
+      // Read https://casl.js.org/v5/en/guide/subject-type-detection#use-classes-as-subject-types for details
```
[the remark this answers is not in the document]
Comment (stalniy) [reply content/security/authorization.md:254]: like this below?

```md
> info **Hint** `detectSubjectType` option let CASL understand how to get subject type out of an object. Read https://casl.js.org/v5/en/guide/subject-type-detection#use-classes-as-subject-types for details
```

[the remark this answers is not in the document]
Comment (kamilmysliwiec) [reply content/security/authorization.md:254]: Correct

[the remark this answers is not in the document]
Comment (Tony133) [reply content/security/authorization.md:254]: yes exact like the one below

File: content/security/authorization.md [hunk 2]
```diff
@@ -260,9 +260,11 @@ export class CaslAbilityFactory {
 
 > warning **Notice** `all` is a special keyword in CASL that represents "any subject".
 
-> info **Hint** `Ability`, `AbilityBuilder`, and `AbilityClass` classes are exported from the `@casl/ability` package.
+> info **Hint** `Ability`, `AbilityBuilder`, `AbilityClass`, and `ExtractSubjectType` classes are exported from the `@casl/ability` package.
 
-In the example above, we created the `Ability` instance using the `AbilityBuilder` class. As you probably guessed, `can` and `cannot` accept the same arguments but has different meanings, `can` allows to do an action on the specified subject and `cannot` forbids. Both may accept up to 4 arguments. To learn more about these functions, visit the official [CASL documentation](https://casl.js.org/v5/en/guide/intro). 
+> info **Hint** `detectSubjectType` option let CASL understand how to get subject type out of an object. Read the official [CASL documentation](https://casl.js.org/v5/en/guide/subject-type-detection#use-classes-as-subject-types) for details.
```
Comment (Tony133) [inline content/security/authorization.md:265]: For more information read

**Review by Tony133 — APPROVED** [review]

**Review by Tony133 — COMMENTED** [review]
It's OK for me

**Review by Tony133 — APPROVED** [review]

**Comment by kamilmysliwiec** [comment]
LGTM thanks!

**Pull request merged**